unity/Assets/Maroon/GlobalEntities/SoundManager/SoundManager.cs

```diff
@@ -118,9 +126,6 @@ private void Awake()
 
             // Keep alive
             DontDestroyOnLoad(this.gameObject);
-
-            // TODO: Cleanup
-            _musicSource = GetComponent<AudioSource>();
```

Comment:
AudioSources are already assigned in prefab via Inspector. Calling GetComponent when multiple Components of a type are available is ambiguous.